<ui version="4.0" >
 <class>SessionDialog</class>
 <widget class="QWidget" name="SessionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>146</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="4" row="0" column="0" >
    <widget class="QListView" name="sessionList" >
     <property name="tabKeyNavigation" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="newButton" >
     <property name="text" >
      <string>New</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="deleteButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Delete</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="activateButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Make Active</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
